beacon-chain/builder/service_test.go

```diff
@@ -37,3 +38,12 @@ func Test_RegisterValidator(t *testing.T) {
 	require.NoError(t, s.RegisterValidator(ctx, []*eth.SignedValidatorRegistrationV1{{Message: &eth.ValidatorRegistrationV1{Pubkey: pubkey[:], FeeRecipient: feeRecipient[:]}}}))
 	assert.Equal(t, true, builder.RegisteredVals[pubkey])
 }
+
+func TestService_PollRelayerStatus_ContextCanceled(t *testing.T) {
```

Comment:
This doesn't test anything, better to just remove

Reply:
You are right. I thought it would be nice for code coverage regardless. Removed :)